### ISSUE TITLE
Jupyter specific commands incorrectly show up for all notebook editors

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode-jupyter#5559]
**Repro**
1. With Jupyter installed 
1. Open a [GitHub issues notebook](https://github.com/microsoft/vscode-github-issue-notebooks)
2. In the command platte, run `Notebook: collapse all notebook cells`

**bug**

<img width="686" alt="Screen Shot 2021-04-19 at 5 46 33 PM" src="https://user-images.githubusercontent.com/12821956/115320677-2e7ab900-a137-11eb-8875-957d9796a774.png">

**Comment by mjbvz** [comment]
I believe you need to use a more specific context key here: https://github.com/microsoft/vscode-jupyter/blob/main/package.json#L1178

It's also confusing that these commands are under the generic `Notebook` command group. It made me think they were coming from VS Code at first

**Comment by DonJayamanne** [comment]
> confusing that these commands are under the generic Notebook command group. It made me think they were coming from VS Code at first

Last we discussed this. We decided to change Jupyter to notebook.
I think it should start as notebook.
Most of the Jupyter users will be using notebook command to create cells, switch kernel,  etc, and those are prefixed `Notebook` having others prefixed `Jupyter` is confusing.

Either it's all consistently Jupyter or notebook, the later is the oblivious choice.
I think hiding the command for other notebooks will address the problem you had.

For users of ipynb (Jupyter) they wouldn't care whether it's a command contributed by vscode or an extension, they just called about notebooks.
